src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/font_style.js

```diff
@@ -2,7 +2,8 @@ define([
   "base/js/namespace",
   "jquery",
   "./font_style/font_control",
-  "./font_style/font_spacing"
+  "./font_style/font_spacing",
+  "./Color_style/Color_control"
 ], function(Jupyter, $, Font_control, Font_spacing) {
```

Comment:
You need to add Colour_control to this line and then create the cc_obj in order for it to work

Reply:
done